CHANGES

```diff
@@ -42,6 +42,10 @@
   *) Support for SSL_OP_NO_ENCRYPT_THEN_MAC in SSL_CONF_cmd.
      [Emilia Käsper]
 
+  *) The RSA "null" method, which was partially supported to avoid patent
+     issues, has been replaced to always returns NULL.
+     [Rich Salz]
```

Comment:
It removes something that doesn't really work; I consider that a bug.  If you don't, then only approve for master when you're happy with this.